packages/playground/src/utils/validators.ts

```diff
@@ -777,3 +777,20 @@ export async function isAvailableName(grid: GridClient, name: string) {
     return { message: "Name is already taken." };
   }
 }
+
+/**
+ * Validates an SMTP input string.
+ *
+ * Checks if the input string is a valid email address and does not contain special characters.
```

Comment:
Is it only email? if so change the input parameter to `email`

Reply:
It now accepts username and email in the same input and it could validate each.